File: Haskell/beginner/1042.hs
-- https://judge.beecrowd.com/pt/problems/view/1042
-- Importing printf
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assigning them to variables
    usr_input <- getLine
    let [a, b, c] = map read (words usr_input) :: [Double]

    -- Sorting values
    if a < b then
        if a < c then do
            printf "%.0f\n" a
            if b < c then do
                printf "%.0f\n" b
                printf "%.0f\n" c
            else do
                printf "%.0f\n" c
                printf "%.0f\n" b
        else do
            printf "%.0f\n" c
            printf "%.0f\n" a
            printf "%.0f\n" b
    else if b < c then do
        printf "%.0f\n" b
        if a < c then do
            printf "%.0f\n" a
            printf "%.0f\n" c
        else do
            printf "%.0f\n" c
            printf "%.0f\n" a
    else do 
        printf "%.0f\n" c
        if a < b then do
            printf "%.0f\n" a
            printf "%.0f\n" b
        else do
            printf "%.0f\n" b
            printf "%.0f\n" a

    do
        printf "\n%.0f\n" a
        printf "%.0f\n" b
        printf "%.0f\n" c
        
-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1042.c
/*https://judge.beecrowd.com/pt/problems/view/1042*/
/*Including libs*/
#include <stdio.h>

/*Main body*/
int main(){
    /*Starting Code and variables*/
    int a, b, c;

    /*Scanning values*/
    scanf("%d %d %d", &a, &b, &c);

    if(a < b && a < c){
        printf("%d\n", a);
        if(b < c){
            printf("%d\n", b);
            printf("%d\n", c);
        } else{
            printf("%d\n", c);
            printf("%d\n", b);
        }
    } else if(b < a && b < c){
        printf("%d\n", b);
        if(a < c){
            printf("%d\n", a);
            printf("%d\n", c);
        } else{
            printf("%d\n", c);
            printf("%d\n", a);
        }
    } else if(c < a && c < b){
        printf("%d\n", c);
        if(a < b){
            printf("%d\n", a);
            printf("%d\n", b);
        } else{
            printf("%d\n", b);
            printf("%d\n", a);
        }
    }

    printf("\n%d\n%d\n%d\n", a, b, c);

    /*Ending Code*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/